src/components/pages/AdminMeetingsDashBoard/CrudModals/CreateModal.js

```diff
@@ -0,0 +1,212 @@
+import React, { useEffect, useState } from 'react';
+import 'antd/dist/antd.css';
+import { Button, Modal } from 'antd';
+import axios from 'axios';
```

Comment:
should import useAxiosWithAuth0 instead of axios

Reply:
This has been fixed. Thank you!